src/index.js

```diff
@@ -12,6 +12,7 @@ import renderPage from './modules/renderPage.js';
 import triggerEvent from './modules/triggerEvent.js';
 import updateTransition from './modules/updateTransition.js';
 
+import pckg from '../package.json';
```

Comment:
Unless I'm mistaken, this will always bundle the whole JSON file, which we likely don't want. 
One of the workarounds for this is to use a global variable and use something like webpacks DefinePlugin to replace it with the actual value from `package.json`.

Reply:
I thought so as well, but interestingly tree-shaking takes care of that nowadays. If you inspect the `dist` folder after running `npm install` and `npm run build` on the PR branch, it's just the version string. Rasso checked with webpack as well, same thing.

Reply:
Damn, we're living in a future. 🤷 
It seems like the webpack is not on the new version tho, so I'm not sure where this update would come from. It used to be a problem as long as I can remember. 
Anyways, you checked, soooo... 👍

Reply:
Actually, I didn't check it with swups webpack version, but with my own v5 webpack (importing `swup/src/index.js` directly, that's how I always test the changes)

Reply:
Good to know, I'll check against webpack 4 as well -- even though we won't be on webpack anymore since this is on the `next` branch.